### DISCUSSION TITLE
Server Side Rendering for the result list

### DISCUSSION THREAD

**Discussion by hungnv-optimizely** [TarekRaafat/autoComplete.js#310] [Q&A] [answered]
Is there any way to show a full list of results that does not have filters?
For example: 
- The keyword: fo
- The result list: [food, foo, abc, def]
- Actual result: [food, foo]
- Expectation result: [food, foo, abc, def]

Please help me if it's possible, thank you.

**Comment by folknor** [comment, marked as answer]
https://tarekraafat.github.io/autoComplete.js/#/configuration?id=searchengine-optional

```js
new autoComplete({
	searchEngine: () => { return true },
})
```

**Reply by hungnv-optimizely** [reply]
thanks, this is the answer that I'm looking for

**Reply by schwarmco** [reply]
stumbled across this answer when i searched for a solution to excatly this problem.

The function changed, so now you can disable the searchengine via 
```javascript
new autoComplete({
    searchEngine: (query, record) => record
})
```
